centos/6/rpmbuilder-c6.bats

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bats
```

Comment:
Is `bats` a widely used tool?

Reply:
I don't think bash testing is very popular, I haven't seen any alternatives to bats but i'm open to suggestions.